src/XrdXrootd/XrdXrootdMonFile.cc

```diff
@@ -174,14 +175,15 @@ void XrdXrootdMonFile::Close(XrdXrootdFileStats *fsP, bool isDisc)
 /*                              D e f a u l t s                               */
 /******************************************************************************/
   
-void XrdXrootdMonFile::Defaults(int intv, int opts, int xfrcnt)
+void XrdXrootdMonFile::Defaults(int intv, int opts, int xfrcnt, int fbuff)
 {
 
 // Set the reporting interval and I/O counter
 //
    repTime = intv;
    xfrCnt  = xfrcnt;
    xfrRem  = xfrcnt;
+   fBuff   = fbuff;
```

Comment:
This isn't going to work at all. If you wind up through the call stack your will notice that Defaults() can be called with fbuff having a value of zero. So this should be:
fBuff = (fbuff <= 0 ? 65472 : fbuff);

Also, why did you call something that is a "size" an fbuff? How about changing it to the more accurate fbsz and fBsz or anything else that tells you its a quantity.

Reply:
I took your suggestion, fbsz and fBsz.  And I added that if statement you suggested.  My pull request would crash when no fbsz is specified in the config without it.